ui/components/multichain/app-header/app-header-unlocked-content.tsx

```diff
@@ -139,15 +139,23 @@ export const AppHeaderUnlockedContent = ({
     origin &&
     origin !== browser.runtime.id;
 
-  const handleMainMenuOpened = () => {
-    trackEvent({
-      event: MetaMetricsEventName.NavMainMenuOpened,
-      category: MetaMetricsEventCategory.Navigation,
-      properties: {
-        location: 'Home',
-      },
+  const handleMainMenuToggle = () => {
+    setAccountOptionsMenuOpen((previous) => {
+      const newValue = !previous;
+      const menuOpen = newValue === true;
+
+      if (menuOpen) {
```

Comment:
```suggestion
      const isMenuOpen = !previous;
      if (isMenuOpen) {
```
Could we refactor to this as
- `newValue = !previous` is just a boolean flip.
- `menuOpen = newValue === true` is redundant because newValue is already a boolean.